airflow/providers/teradata/hooks/teradata.py

```diff
@@ -29,6 +30,7 @@
 from airflow.exceptions import AirflowProviderDeprecationWarning
 from airflow.providers.common.sql.hooks.sql import DbApiHook
 
+T = TypeVar("T")
```

Comment:
What is the use of this?

Reply:
Its not required. Removed it. Thank you.